pkgs/applications/editors/vim/plugins/nvim-treesitter/generated.nix

```diff
@@ -172,7 +172,7 @@
       nativeBuildInputs = [ nodejs tree-sitter ];
       postFetch = "pushd $out && tree-sitter generate && popd";
       rev = "c2fbf21bd3aa45495fe13247e040ad5815250032";
-      hash = "sha256-JOCS72Ux1FMzf6kBVO7aqHNwJ0s3xkhzIFFV9GjJVOs=";
+      hash = "sha256-0Nqfg/SiBNj6VrL3npm7LFsujR2dVKg+SNIjDtjToN4=";
```

Comment:
looks like there might be a bug in the update script, do you mind reverting the ones that are only hash changes?

Reply:
Done :)